### PR TITLE
kubeone list-images subcommand

### LINKED ISSUE #1277: Images list subcommand
**What feature would you like to be added?**

`kubeone list-images` should reveal the images list that will be used in the given deployment, that take all the configurations in the account, including images from the addons.

**What are use cases of the feature?**
At the moment we have static image-loader script which doesn't have complete images list and we have to maintain it manually.

### PR DESCRIPTION
**Pull request by kron4eg**
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1277

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
kubeone list-images subcommand
```

### CODE REVIEW

File: pkg/cmd/list-images.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		},
+	}
+
+	return cmd
+}
+
+func listImages(opts *listImagesOpts) error {
+	var imgopts []images.Opt
+
+	configBuf, err := os.ReadFile(opts.ManifestFile)
```
Comment (xmudrii) [inline pkg/cmd/list-images.go:60]: We should add a comment briefly describing why we don't use the existing function for parsing the manifest. I guess the reason is that the existing function automatically validates the manifest, which we don't need here at all.

**Review by xmudrii — COMMENTED** [review]
Two nits, but otherwise LGTM. Also, please address the lint error.

File: pkg/cmd/list-images.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	ManifestFile string `longflag:"manifest" shortflag:"m"`
+}
+
+func listImagesCmd(rootFlags *pflag.FlagSet) *cobra.Command {
+	opts := &listImagesOpts{}
+
+	cmd := &cobra.Command{
+		Use:     "list-images",
+		Short:   "List images that will be used",
+		Example: `kubeone list-images -m mycluster.yaml -t terraformoutput.json`,
```
[the remark this answers is not in the document]
Comment (kron4eg) [reply pkg/cmd/list-images.go:42]: 👍

File: pkg/cmd/list-images.go (hunk 1 above)
Comment (kron4eg) [reply pkg/cmd/list-images.go:60]: 👍 
Yep, you are exactly right. Existing mechanism to "load" the config would also validate it and basically I'm forced to ether write full config manually for just image-list, or eve worse to create actual instances with terraform :D

**Review by xmudrii — APPROVED** [review]

**Review by xmudrii — APPROVED** [review]

**Pull request merged**